yarn_api_client/base.py

```diff
@@ -66,6 +66,9 @@ def request(self, api_path, method='GET', **kwargs):
         else:
             headers = {"Content-Type": "application/json"}
 
+        if 'headers' in kwargs and kwargs['headers']:
```

Comment:
Should this block only occur in the `else` condition above?  If not, then we'd need to handle the possibility that `headers` is `None`.

Reply:
Good point, I'll cover the `None` case shortly.

Reply:
I've defaulted headers to `{}`. Hopefully that should behave as is now.